io.vikunja.Desktop.yml

```diff
@@ -0,0 +1,42 @@
+app-id: io.vikunja.Desktop
+app-version: '0.21.0'
+version-hash: '46bd28e13d35f78bbcf2d3b44307059b195f5d1eefcd08937645888f008365e3'
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+command: vikunja-desktop
+
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --env=LD_LIBRARY_PATH=/app/lib
+  - --env=ZYPAK_SANDBOX_FILENAME=chrome-sandbox
+  
+modules:
+  - name: vikunja
+    buildsystem: simple
+    build-commands:
+      - mkdir /app/bin/extra /app/lib/extra
+      - mv *.so* /app/lib/
+      - install -D vikunja.sh "/${FLATPAK_DEST}/bin/vikunja"
+      - mv * /app/bin/extra/
+    sources:
+      - type: archive
+        url: https://dl.vikunja.io/desktop/{{ app-version }}/Vikunja%20Desktop-{{ app-version }}.tar.gz
+        sha256: {{ version-hash }}
```

Comment:
neither of these will work. this is not a valid sha256.

Reply:
I thought this would get expanded from the variables defined at the beginning of this file?

Reply:
where in the dcoumentation does it say that? (so I can actually fix the documentation)

Reply:
I run the hash check (`sha256sum Vikunja\ Desktop-0.21.0.tar.gz`) locally directly after downloading the file (https://dl.vikunja.io/desktop/0.21.0/Vikunja%20Desktop-0.21.0.tar.gz) and that was the hash I got

Reply:
And the variable interpolation?

Reply:
I mean the `{{ version-hash }}` instead of putting the hash there directly. There was a variable `version-hash` at the top level of the yml, I assume that was meant to interpolate the usage in `{{`? Where did you get that from?

Reply:
For accessibility, I set the hash and the app version to 2 separate variables at the start of the file so that they can be easier and better updated when an update arrives

Reply:
I like the idea, but it looks like that won't work because flatpack does not support expanding variables.

Reply:
Yeah, wherever you picked it from, it's not in YAML spec. You could use anchors for that, but there's no interpolation.